docs/api-reference/endpoints/pki/syncs/azure-key-vault/remove-certificates.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Remove Certificates from Azure Key Vault"
+openapi: "POST /api/v1/pki/syncs/azure-key-vault/{pkiSyncId}/remove"
```

Comment:
Thoughts on having this as "DELETE /api/v1/pki/syncs/azure-key-vault/{pkiSyncId}/certificates" instead? I think that follows REST more